drucker-python/README.md

```diff
@@ -4,5 +4,27 @@
 
 ## Official docker image
 ```
-docker pull druckerorg/drucker:python-latest
-```
\ No newline at end of file
+docker pull rekcurd/rekcurd:python-latest
+```
+
+## Entrypoint
+```bash
+#!/usr/bin/env bash
+# You have to set env. Env can be set from Kubernetes Pod boot script.
+#   DRUCKER_SERVICE_GIT_URL="https://github.com/yourproject/yourgit.git"
+#   DRUCKER_SERVICE_GIT_BRANCH="master"
+#   DRUCKER_SERVICE_BOOT_SHELL="start.sh"
+
+
+ECHO_PREFIX="[drucker-settings]: "
+echo "$ECHO_PREFIX Start.."
+
+
+set -e
+set -u
+
+# git clone your-drucker.
+git clone --depth 1 --recursive -b ${DRUCKER_SERVICE_GIT_BRANCH} ${DRUCKER_SERVICE_GIT_URL} /usr/local/src/drucker
+cd /usr/local/src/drucker
+sh ${DRUCKER_SERVICE_BOOT_SHELL}
+```
```

Comment:
In my opinion, writing the content of exiting files on README is redundant, and this makes it hard to manage the file contents and the consistency while it is easier for readers to check the content.
So it would be better to just put the link to the entrypoint.sh.

Reply:
@yuki-mt Thank you.
I have fixed it now. Could you check it again?